src/bosh.js

```diff
@@ -767,7 +767,9 @@ Strophe.Bosh.prototype = {
                 // expanding retry window
                 var backoff = Math.min(Math.floor(Strophe.TIMEOUT * this.wait),
                                        Math.pow(req.sends, 3)) * 1000;
-                setTimeout(sendFunc, backoff);
+                setTimeout(function() {
+                    sendFunc();
+                }, backoff);
```

Comment:
Why did you make this change? Seems like an unnecessary nesting of a function call.